worker/echo/main.go

```diff
@@ -22,6 +24,12 @@ func echo(w *worker.Worker, addr string, id string, responseTo string, metadata
 		return fmt.Errorf("cannot call EmitEvent: %w", err)
 	}
 
+	// Sleep time between receiving the message and sending it
+	if sleepTime > 0 {
+		log.Tracef("sleeping: %d s", sleepTime)
```

Comment:
This could probably be logged at a lower level, like `Info`. Since it will only ever occur when the user invokes `echo` with a value other than the default, having some console feedback might be acceptable.